.github/workflows/ci.yml

```
@@ -178,6 +215,11 @@ jobs:
 
       - name: cargo nextest
         run: cargo nextest run --workspace --locked --all-features --no-fail-fast
+        env:
+          # We are experimenting with git lfs, and we don't expect to run out of bandwidth.
```

Maybe we should reword this slightly. Something like:
```suggestion
          # Although GitHub's LFS limits are very low, we don't expect to run out of bandwidth.
```